### PR TITLE
fix: sql-exception causes session-delete (back-button)

### PR DESCRIPTION
**Pull request by SeduroDotCom**

### CODE REVIEW

File: spec/features/meta_keys_spec.rb
Comment (nimaai) [inline spec/features/meta_keys_spec.rb:1]: you forgot `binding.pry`

File: spec/features/meta_keys_spec.rb [hunk 1]
```diff
@@ -567,6 +567,9 @@ def meta_datum_types
         expect(current_path).to eq(meta_key_path(meta_key))
         expect(meta_key.meta_data.reload).to eq(meta_data)
 
+        visit meta_key_path("")
```
Comment (nimaai) [inline spec/features/meta_keys_spec.rb:570]: does this work? normally one should pass the meta_key to this path helper.

**Review by nimaai — CHANGES_REQUESTED** [review]

File: spec/features/meta_keys_spec.rb (hunk 1 above)
Comment (SeduroDotCom) [reply spec/features/meta_keys_spec.rb:570]: Yes, fnc only adds meta_key to url `/admin/meta_keys/ `
Both scenarios have been tested with binding.pry - with/out fix

Comment (SeduroDotCom) [reply spec/features/meta_keys_spec.rb]: Replaced it by string-definition

**Pull request merged**